ui.tests/test-module/specs/replaceaction/replacelayout-container.authoring.spec.js

```diff
@@ -79,14 +80,14 @@ describe('component replace - Authoring', function () {
         // Check If Dialog Options Are Visible
         if (componentEditPathSelector == buttonEditPathSelector) {
             cy.get(textInput)
-                .should("exist");
+                .should("not.exist");
             cy.get(submitButton)
                 .should("exist");
-            cy.get(textInput)
```

Comment:
@egmox , It seems like we have tested for button and panels , but what about all other input type components and non input type components that we have, can we write a generic test case to check replace behaviour for them also.
cc:- @nit23uec , what do you suggest

Reply:
@deepprakash345 In the frontend, we don't have any parameters that tells if it is input type or not. We will need to check list of all components if it is showing in dialog or not